pages/products/index.js

```diff
@@ -10,9 +10,8 @@ function FrontMatter() {
     <Col>
       <p className="text-black">
         Products are the solutions we build for our customers, including Knack
-        apps, AMANDA apps, custom software, and data systems. We improve and
-        extend our products over time so that they deliver continuous value to
-        Austin Transportation as business needs evolve.
+        apps, AMANDA apps, custom software, and data systems. We improve and extend our products over time so that 
+        they deliver continuous value to Transportation and Public Works staff as business needs evolve.
```

Comment:
putting in my two cents that the phrase "AMANDA apps" doesn't make sense to me. i also kinda don't want to assume people know what "Knack" is. this might be a minor improvement?

> Products are the solutions we build and support for our customers. We improve and extend our products over time so that they deliver continuous value to Transportation and Public Works staff as business needs evolve.

and longer term, it might be nice to expand on this to talk more about our product ops. maybe link out to our intake form, explain our prioritization process, etc cc @amenity

Reply:
Thank you, @johnclary - I'd completely overlooked this... TIme to pull this out of the icebox: [Re-work front-matter on "Products" and "Projects" pages](https://github.com/cityofaustin/atd-data-tech/issues/4806#top) #4806